tools/clang-tidy-plugin/test/use-localized-sorting.cpp

```diff
@@ -14,7 +14,7 @@ bool operator<( const NonString &, const NonString &rhs ) noexcept;
 bool f0( const std::string &l, const std::string &r )
 {
     return l < r;
-    // CHECK-MESSAGES: warning: Raw comparison of 'const std::string' (aka 'const basic_string<char>').  For UI purposes please use localized_compare from translations.h. [cata-use-localized-sorting]
+    // CHECK-MESSAGES: warning: Raw comparison of 'const std::string' (aka '{{.*basic_string.*}}').  For UI purposes please use localized_compare from translations.h. [cata-use-localized-sorting]
```

Comment:
it's `aka 'const basic_string<char, char_traits<char>, allocator<char>>'` on windows